app/Exceptions/Handler.php

```diff
@@ -39,14 +40,31 @@ public function report(Exception $exception)
     /**
      * Render an exception into an HTTP response.
      *
-     * @param \Illuminate\Http\Request $request   the app request
-     * @param \Exception               $exception the exception that needed to be redered
+     * @param \Illuminate\Http\Request $request the app request
+     * @param \Exception               $e       the exception that needed to be redered
      *
      * @return \Illuminate\Http\Response
      */
-    public function render($request, Exception $exception)
+    public function render($request, Exception $e)
     {
-        return parent::render($request, $exception);
+        $viewName = $this->getErrorPage404();
```

Comment:
the `$viewName` variable is only use when the `$e` is the instance of `ModelNotFoundException`, so if you put the variable on the top of the function it will be not good.